oxd-server/src/test/java/org/xdi/oxd/server/GetTokensByCodeTest.java

```diff
@@ -30,9 +29,8 @@
     public void test(String host, String opHost, String redirectUrl, String userId, String userSecret) throws IOException {
         ClientInterface client = Tester.newClient(host);
         final RegisterSiteResponse site = RegisterSiteTest.registerSite(client, opHost, redirectUrl);
-        GetTokensByCodeResponse tokensResponse = tokenByCode(client, site, userId, userSecret, CoreUtils.secureRandomString());
+        GetTokensByCodeResponse tokensResponse = whenValidCodeIsUsed_shouldGetTokenInResponse(client, site, userId, userSecret, CoreUtils.secureRandomString());
```

Comment:
revert it back to `tokenByCode`. I meant test should be renamed `test` -> `whenValidCodeIsUsed_shouldGetTokenInResponse`